docs/generalqa.md

```diff
@@ -34,6 +34,39 @@ To contribute to this section, please review our [Code of Conduct](CODE_OF_CONDU
 
 ---
 
+## SMTP Configuration
+**Q:** How can I enable SMTP for Archipelago?
+
+**A:** For standard demo deployments, SMTP is not setup to send emails. To enable SMTP:
+
+1. Run the following commands in your terminal.
+
+```Shell
+docker exec -ti esmero-php bash -c 'php -dmemory_limit=-1 /usr/bin/composer require drupal/smtp:^1.0'
+docker exec -ti esmero-php bash -c 'drush en -y smtp'
+```
+
+2.) Check that the SMTP module has been enabled by navigating (as admin user) to the EXTEND module (`localhost:8001/admin/modules`). You should see "SMTP Authentication Support" listed.
+
+3.) Navigate to `localhost:8001/admin/config/system/smtp` to configure the SMTP settings.
```

Comment:
all this 3.) need to be changed to Markdown numeric lists  e.g `3.`

Reply:
Ah right, all changed now. Sorry about that!